test/node/helpers.js

```diff
@@ -1,12 +1,12 @@
-var { expect } = require('chai');
-var constants = require('../../src/core/constants.js');
+import expect from 'chai';
+import constants from '../../src/core/constants.js';
```

Comment:
this is how we import all the constants into one object, since each of the individual values are exported individually
```suggestion
import * as constants from '../../src/core/constants.js';
```